src/Views/AppListUpdateView.vala

```diff
@@ -54,7 +54,7 @@ namespace AppCenter.Views {
             update_mutex = GLib.Mutex ();
             apps_to_update = new Gee.LinkedList<AppCenterCore.Package> ();
 
-            sc = new SuspendControl ();
+            sc = SuspendControl.get_default ();
```

Comment:
It isn't needed anymore to hold a reference to `SuspendControl`, just use `SuspendControl.get_default ()` in all places where that's needed.